docs/Updating-npm-bundled-node-gyp.md

```diff
@@ -1,63 +1,41 @@
 `npm` bundles its own, internal, copy of `node-gyp`. This internal copy is independent of any globally installed copy of node-gyp that
 you may have installed via `npm install -g node-gyp`.
 
-This means that while `node-gyp` doesn't get installed into your `$PATH` by default, npm still keeps its own copy to invoke when you
-attempt to `npm install` a native add-on.
+Generally, npm's library files are installed inside your global "node_modules",where npm is installed (run `npm prefix` and add `lib/node_modules`, or just `node_modules` for Windows). There are some exceptions to this. Inside this global `node_modules/`there will be an `npm/` directory and inside this you'll find a `node_modules/node-gyp/` directory. So it may look something like `/usr/local/lib/node_modules/npm/node_modules/node-gyp/`. This is the version of node-gyp that ships with npm.
```

Comment:
```suggestion
Generally, npm's library files are installed inside your global "node_modules", where npm is installed (run `npm prefix` and add `lib/node_modules`, or just `node_modules` for Windows). There are some exceptions to this. Inside this global `node_modules/` there will be an `npm/` directory and inside this you'll find a `node_modules/node-gyp/` directory. So it may look something like `/usr/local/lib/node_modules/npm/node_modules/node-gyp/`. This is the version of node-gyp that ships with npm.
```

Reply:
On windows global `node_modules` and *node's* `node_modules` are two diffrent dirs, aren't they? Thats why 
> Generally, npm's library files are installed inside your global "node_modules", where npm is installed

can really mislead.

I propose this:
> So npm's internal copy of node-gyp isn't stored inside *global* `node_modules` and thus isn't available for use as a standalone package. Npm uses it's internal copy of node-gyp to automatically build native addon when you are trying to it install via `npm install`